auctions/templates/bid.html

```diff
@@ -14,31 +14,24 @@
          {% endif %}
          <input name="url" type="hidden" value="{{ url }}"></input>
          <fieldset>
-            <div class="pure-u-2-5">
+            <div class="pure-group">
                <label for="ask">Asking</label>
-            </div>
-            <div class="pure-u-2-5">
-               <label for="offer">Offering</label>
-            </div>
-            <div class="pure-u-1-5"></div>
-            <div class="pure-u-2-5">
-               <input name="ask"
+               <input id="ask"
```

Comment:
Crap, I already merged this, but the `name` attribute is required to submit the data during the POST. I'll send a PR to fix.